graph/multigraph.go

```diff
@@ -24,7 +24,7 @@ type Multigraph interface {
 	Has(id int64) bool
 
 	// Node returns the node with the given ID if it exists
```

Comment:
For consistency with `// Node returns the node in the graph with the given ID if it exists, and nil otherwise.`

```diff
- // Node returns the node with the given ID if it exists in the multigraph, and nil otherwise.
+// Node returns the node in the multigraph with the given ID if it exists, and nil otherwise.
```

Actually, come to think of it. I prefer the way multigraph was stated. I.e. `Node returns the node with the given ID if it exists in the multigraph, and nil otherwise`.

So, for consistency, either update the multigraph comment, or better yet (in my opinion), the other comments to `// Node returns the node with the given ID if it exists in the graph, and nil otherwise.`

Reply:
Done latter.